src/models/login-model.ts

```diff
@@ -0,0 +1,6 @@
+type LoginModel = {
```

Comment:
We have 4 models that seems represent the model User. We could talk about it on our next call but I believe we could use just one or we also could use extends because those 4 models have email for example.